packages/salesforcedx-vscode-apex-replay-debugger/package.json

```diff
@@ -24,6 +24,9 @@
     "Debuggers"
   ],
   "dependencies": {
+    "@salesforce/apex-node": "0.1.4",
```

Comment:
let's also make sure to update the apex-node version to the one that includes the auth issue fix so we don't run into that while running tests here